chassis/invokers/src/main/java/com/griddynamics/jagger/invoker/http/v2/JHttpQuery.java

```diff
@@ -0,0 +1,372 @@
+package com.griddynamics.jagger.invoker.http.v2;
+
+import org.springframework.http.HttpHeaders;
```

Comment:
The idea was to hide underneath implementation, but here we expose Spring's guts. Does not look consistent.